Documentation/joss/paper.md

```diff
@@ -1,15 +1,5 @@
-
-Note to OIP developers: observe the [guidelines](https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain) and our [notes](./notes.md).
-
-Most important instructions:
-
-> 250-1000 words
-> explain the software functionality and domain of use to a *non-specialist* reader
-
-See [example](http://bit.ly/2x22gxT) and [a more complex example](https://joss.readthedocs.io/en/latest/submitting.html#example-paper-and-bibliography).
-
 ---
-title: 'TUM OpenInfraPlatform: open source package for simultaneous viewing and analysis of Industry Foundation Classes (IFC) and Point Cloud Data (PCD) models'
+title: 'TUM Open Infra Platform: open source package for simultaneous viewing and analysis of digital models in the civil engineering domain'
 tags:
```

Comment:
What about BIM? Building Information Modelling? to obvious? ;)

Reply:
resolved in 327d66e20deef940696b0540e3ee53d9996b1643